server/svix-server/src/v1/endpoints/endpoint.rs

```diff
@@ -170,6 +170,60 @@ struct RecoverIn {
     since: DateTime<Utc>,
 }
 
+#[derive(Clone, Debug, PartialEq, Validate, Deserialize)]
+#[serde(rename_all = "camelCase")]
+struct EndpointHeadersIn {
+    #[validate]
+    headers: EndpointHeaders,
+}
+
+#[derive(Clone, Debug, PartialEq, Serialize)]
+#[serde(rename_all = "camelCase")]
+struct EndpointHeadersOut {
+    headers: HashMap<String, String>,
```

Comment:
Same comment as elsewhere, why not use the `http::header` types?

Reply:
Leaving this for now, see my previous comment.